docker/start.py

```diff
@@ -122,7 +118,12 @@ def generate_config_from_template(config_dir, config_path, environ, ownership):
             config_dir,
             "--generate-keys",
         ]
-    )
+
+    if ownership is not None:
```

Comment:
please can you update the docstring to say that `ownership` can be `None`, and what that means?